src/main/java/pl/exsio/querydsl/entityql/entity/scanner/NativeQEntityScanner.java

```diff
@@ -0,0 +1,61 @@
+package pl.exsio.querydsl.entityql.entity.scanner;
+
+import pl.exsio.querydsl.entityql.entity.metadata.QEntityColumnMetadata;
+import pl.exsio.querydsl.entityql.entity.metadata.QEntityMetadata;
+import pl.exsio.querydsl.entityql.entity.scanner.nativesql.Column;
+import pl.exsio.querydsl.entityql.entity.scanner.nativesql.NamingStrategy;
+import pl.exsio.querydsl.entityql.entity.scanner.nativesql.Table;
+import pl.exsio.querydsl.entityql.ex.MissingIdException;
+
+import java.util.List;
+
+/**
+ * QEntityScanner implementation based on Spring Data JDBC.
+ */
+public class NativeQEntityScanner implements TableScanner {
```

Comment:
Is it Native though? I'd call it "Dynamic"

Reply:
correction: "NativeQEntityScanner" -> "RuntimeTableScanner"

Reply:
renamed to RuntimeTableScanner